assets/_sass/custom-viz/boeing/_base.scss

```diff
@@ -24,14 +24,31 @@
   max-width: 700px;
 
   p {
-    min-height: 100vh;
     margin: 0;
     text-align: left;
+    background-color: rgba(255, 255, 255, 0.924);
+    padding: 1.5rem;
 
     &:not(:last-of-type) {
-     margin: 0 0 1rem;
+     margin: 0 0 40rem;
     }
   }
+
+  .part {
+    margin: 0.1rem;
+    background-color: $yellow;
+
+    &:hover {
+      background-color: $dark-yellow;
+      transition: all 0.2s ease-in-out;
+      cursor: pointer;
+    }
+  }
+}
+
+.timer {
```

Comment:
Be careful about prefixing/limiting the specificity of styles for the interactive. You don't want them to accidentally interfere with other items on the page.